.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  "extends": [
+    "react-app",
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
+  ],
+  "settings": {
+    "react": {
+      "version": "999.999.999"
+    }
+  }
```

Comment:
the eslint in protocol is a lot different than this, im ok with not matching that but @chrismaree might have a different opinon